openedx/features/marketplace/constants.py

```diff
@@ -0,0 +1,33 @@
+ORGANIZATION_SECTOR_CHOICES = (
+    ('health', 'Health & Well-being'),
```

Comment:
Becuase keys will be saved in data base I guess we should include max info in the keys because they will be using this data for stats, first one could be `helth_and_well_being` then if we see third one equality and gender equality are different things so should change it to `gender_equality` and up to so on.

Reply:
similarly in ORG_PROBLEM_CHOICES justice and social justice are different things similarly equality and gender equality are different, so let's please revisit all choices and add max info in keys.

Reply:
@ahmed-faraz46 I am updating the string but making longer string will cause more comparisons. But that's fine as we are agreed to write more readable strings.